### DISCUSSION TITLE
Checkbox with `Indeterminate` option does not adequetly offer the same functionality as the `TriStateCheckbox` from v3

### DISCUSSION THREAD

**Discussion by OliverRC** [primefaces/community#2957] [PrimeVue] [answered]
For example, with `<Checkbox v-model="checked" indeterminate binary />`  you can only toggle between checked `true`/`false` there is no way to get it back to `indeterminate` or a 'nothing state' like you could with the `TriStateCheckbox` from v3.

With the `TriStateCheckbox` you could toggle between all three states which was perfect for filtering in `DataTables`.

![TriStateCheckbox](https://github.com/user-attachments/assets/4e49891e-3121-4ac4-af59-77c11d4fb088)
![checkbox-inderterminate](https://github.com/user-attachments/assets/acfedc75-c716-48c2-b7e1-dfcc4e8cb91b)

I therefore disagree with the migration advice 

> TriStateCheckbox | Use Checkbox with indeterminate option.

**Comment by j0rgedev** [comment, marked as answer]
Hello @OliverRC. This question was also asked when v4 was just released. This guy offered a solution to handle the new checkbox as the old tristatecheckbox. He provided the repro: https://stackblitz.com/edit/ghfhbr. 
[Related discussion](https://github.com/orgs/primefaces/discussions/2463)
